docs/main/Solutioning/openebs-on-kubernetes-platforms/gke.md

```diff
@@ -0,0 +1,299 @@
+$ kubectl mayastor get block-devices gke-gke-ssd-default-pool-2a0f964a-hv99
+ DEVNAME       DEVTYPE  SIZE    AVAILABLE  MODEL       DEVPATH                                              MAJOR  MINOR  DEVLINKS 
+ /dev/nvme1n1  disk     375GiB  yes        nvme_card0  /devices/pci0000:00/0000:00:04.0/nvme/nvme1/nvme1n1  259    0      "/dev/disk/by-id/google-local-nvme-ssd-0", "/dev/disk/by-id/nvme-nvme.1ae0-6e766d655f6361726430-6e766d655f6361726430-00000001", "/dev/disk/by-id/nvme-nvme_card0_nvme_card0", "/dev/disk/by-id/nvme-nvme_card0_nvme_card0_1", "/dev/disk/by-path/pci-0000:00:04.0-nvme-1" 
+```
+
+The block size of the disks should be specified by the block size of the local SSD in your GKE. Run the following commands from the worker node to find the SSD block size:
+
+**Command**
+
+```
+fdisk -l /dev/nvme1n1
+```
+
+**Output**
+
+```
+root@gke-gke-ssd-default-pool-2a0f964a-980h:~# fdisk -l /dev/nvme1n1
+Disk /dev/nvme1n1: 375 GiB, 402653184000 bytes, 98304000 sectors
+Disk model: nvme_card0
+Units: sectors of 1 * 4096 = 4096 bytes
+Sector size (logical/physical): 4096 bytes / 4096 bytes
+I/O size (minimum/optimal): 4096 bytes / 4096 bytes
+```
+
+**Command**
+
+```
+lsblk -o NAME,PHY-SeC
+```
+
+**Output**
+
+```
+root@gke-gke-ssd-default-pool-2a0f964a-980h:~# lsblk -o NAME,PHY-SeC
+NAME         PHY-SEC
+nvme0n1          512
+├─nvme0n1p1      512
+├─nvme0n1p14     512
+└─nvme0n1p15     512
+nvme1n1         4096
+root@gke-gke-ssd-default-pool-2a0f964a-980h:~# 
+```
+
+### Pool.yaml
+
+Create a pool with the following pool.yaml:
+
+```
+apiVersion: "openebs.io/v1beta2"
+kind: DiskPool
+metadata:
+  name: POOL_NAME
+  namespace: mayastor
+spec:
+  node: NODENAME
+  disks: ["aio:////dev/disk/by-id/google-local-nvme-ssd-0?blk_size=4096"]
+```
+
+**Command**
+
+```
+kubectl apply -f pool.yaml
+```
+
+**Output**
+
+```
+diskpool.openebs.io/pool-1 created
+```
+
+## Configuration
+
+- See [here](../rs-configuration.md#create-replicated-pv-mayastor-storageclasss) for instructions regarding StorageClass creation.
+
+- See [here](../rs-deployment.md) for instructions regarding PVC creation and deploying an application.
+
+## Node Failure Scenario
+
+The GKE worker nodes are a part of a managed instance group. A new node is created with a new local SSD if a node becomes unreachable or faulty. In such cases, recreate the pool with a new name. Once the new pool is created, the OpenEBS Replicated PV Mayastor will rebuild the volume with the replicated data.
+
+:::note
+When a node gets replaced with a new node, all the node labels and huge pages configurations will be lost. You must reconfigure these [prerequisites](#prerequisites) on the new node. 
+:::
+
+**Example**
+
+When the node `gke-gke-local-ssd-default-pool-dd2b0b02-8twd` is rebooted, it fails. Subsequently, a new node/disk is acquired, resulting in the pool-4 being classified as unknown and the Replicated PV Mayastor volume being classified as degraded due to the failure of one of the replicas.
```

Comment:
"Pool-3" as we changed the sequence. 

When the node `gke-gke-local-ssd-default-pool-dd2b0b02-8twd` is **_failed._** Subsequently, a new node/disk is acquired, resulting in the pool-3 being classified as unknown and the Replicated PV Mayastor volume being classified as degraded due to the failure of one of the replicas.